### DISCUSSION TITLE
Looking for beta testers

### DISCUSSION THREAD

**Discussion by Panda2340** [community/community#134796] [Other Feature Feedback, Questions, & Ideas]
Hi,
I am looking for beta testers who would like to test my upcoming operating system for the Raspberry Pi Pico. If you don't have a Raspberry Pi Pico, No problem! The builds for beta testers are supposed to run in a terminal.

Criteria:
- At least basic experience with Python
- A computer with a terminal that can run Python scripts
- Python installed on your computer, if you don't have it installed, use one of the links below to download:
Download Source Release: https://www.python.org/downloads/source/
Windows Download: https://www.python.org/downloads/windows/
Mac Download: https://www.python.org/downloads/macos/
Get Python for other platforms: https://www.python.org/download/other/

If you want to be a beta tester, feel free to become one! Just ask me by commenting on this post.

If you are a beta tester, you can give feedback about the OS here: https://github.com/Panda2340/inspireos-beta/discussions/categories/beta-testers

Updates about the OS will be posted here: https://github.com/Panda2340/inspireos-beta/discussions/categories/announcements

Bye! :)

**Comment by ebndev** [comment]
Hi @Panda2340, thanks for sharing with the community!

Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
